src/components/calendar/Calendar.js

```diff
@@ -8,9 +7,7 @@ export const Calendar = ({ events }) => {
   if (events.length) {
     eventList = events.map(event => {
       return (
-        <Link to={`/event/${event.id}`} key={event.id} >
```

Comment:
Ooops! I see that this became problematic for onClick completed functionality. Nice refactor!